### PR TITLE
Merge sort and calculator

### PR DESCRIPTION
**Pull request by Yevhenii-Khaliuk**

### CODE REVIEW

**Review by BondarenkoJek — CHANGES_REQUESTED** [review]
Good job!
But try in calculator this expression (-1 - 1) --> do not work, should fix this mistake

File: src/calculator/Calculator.java [hunk 1]
```diff
@@ -84,4 +112,6 @@ private static double performOperator(char operator, double rightNum, double lef
                 return 0;
         }
     }
+
```
Comment (BondarenkoJek) [inline src/calculator/Calculator.java]: delete lines

**Review by BondarenkoJek — APPROVED** [review]
LGTM

File: src/calculator/Calculator.java (hunk 1 above)
Comment (Yevhenii-Khaliuk) [reply src/calculator/Calculator.java]: should I leave default operator empty?

**Pull request merged**